semantics/executable-spec/src/Data/MemoBytes.hs

```diff
@@ -66,6 +75,9 @@ instance Show t => Show (MemoBytes t) where show (Memo y _) = show y
 
 instance Ord t => Ord (MemoBytes t) where compare (Memo x _) (Memo y _) = compare x y
 
+instance HasField tag t c => HasField (tag::Symbol) (MemoBytes t) c where
+   getField (Memo x _) = getField @tag x
```

Comment:
This is really nice :-)

Reply:
Works nicely in MemoBytes, not so nicely elsewhere.